SPECS/python3/python3.spec

```diff
@@ -17,7 +17,6 @@ Source0:        https://www.python.org/ftp/python/%{version}/Python-%{version}.t
 # It has been removed in Python-3.12.0.tar.xz, but as our packages still require it, we will still provide for now.
 Source1:        https://github.com/python/cpython/blob/3.9/Tools/scripts/pathfix.py
 Patch0:         cgi3.patch
-Patch1:         CVE-2023-6597.patch
```

Comment:
It seems we missed to `git rm` this file.

Reply:
Thanks for catching it. I have remove the file from the git. Please check the latest commit.